docs/mermaid/polkadot_sdk.mmd

```diff
@@ -0,0 +1,26 @@
+flowchart LR
+    subgraph Parachain[A Polkadot Parachain]
+        ParachainCollator[Parachain Collator]
+        ParachainRuntime[Parachain Runtime]
+    end
+
+    subgraph Polkadot[The Polkadot Relay Chain]
```

Comment:
Perhaps could be simpler renaming 
- "A Substrate-based blockchain" -> "A standalone blockchain (e.g. the Polkadot Relay Chain)" (they are all substrate-based)
- "A Polkadot Parachain" -> "A parachain"

and removing "The Polkadot Relay Chain" because it's more or less a duplicate of the "A Substrate-based blockchain" diagram?

Reply:
True, but at the same time I think there is some value in saying "pss, Polkadot Relay Chain is really just a (almost) normal substrate-based chain"?

Reply:
I did not change this for now, see outcome with 1896ac74b52b5302671ba82f0852819945b9c531 and lmk what you think now.